extensions/resource-deployment/src/ui/deploymentInputDialog.ts

```diff
@@ -110,7 +110,7 @@ export class DeploymentInputDialog extends DialogBase {
 				});
 			}
 		} else {
-			vscode.commands.executeCommand(this.dialogInfo.command, model);
+			await vscode.commands.executeCommand(this.dialogInfo.command, model);
```

Comment:
Next time it'd be good to split fixes like these up into a separate PR so it's easier to review both parts.